Cloud Application Manager/Automating Deployments/deploymentpolicy-box.md

```diff
@@ -89,6 +87,26 @@ Give access to other individuals or team workspaces by clicking on the pencil of
 
 For more information about sharing, see [sharing boxes, instances and providers](../Core Concepts/workspaces-and-collaboration.md#sharing-boxes-instances-and-providers).
 
+### Deployment policy variables
+
+Clicking the *Code* section of a Deployment Policy box gives the user the possibility of editing the already defined Policy, as well as defining some variables to parametrize that box.
+
+![Available variable types for Deployment Policy boxes](../../images/cloud-application-manager/deploymentpolicyboxes-variables.png)
```

Comment:
There is a folder for deployment-policy images, please move this one there

Reply:
All images of this article are stored in cloud-application-manager folder. Should I move all of them to deployment-policy?

Reply:
Already moved 👍